### DISCUSSION TITLE
Deletion of entities

### DISCUSSION THREAD

**Discussion by valamercery** [geovistory/.github#5] [Ideas]
Discussion started by @tla Oct 24

**Is your feature request related to a problem? Please describe.**
Now that ongoing namespaces are hidden by default, would it be possible to implement deletion of classes and properties? In any event, I don't see any way to do this in the current interface and there is no hint in the user guide that it is possible.

**OntoME username (optional):**
tla

**Comment by iltzortz** [comment]
I have a similar issue

I created in ontome a class that has been assinged an identifier that i want to change. i can define a new label for the class but i cannot change the identifier. So i was thinking that there must be a way to delete it and create a new one. 

So is Class deletion supported? Likewise, is project deletion supported? i cannot find a way to do this in the current interface

**Comment by valamercery** [comment]
Class and property deletion are not supported at the moment. However, a function that allows a class to be made obsolete is currently under development. If there is no documentary interest in keeping a non-validated entity, it can be modified to replace it with another. 

Identifiers can be changed by switching to manual mode in the "Identifiers" tab of the root namespace.

Anyway, this discussion could help us determine whether or not it would be appropriate to add a class/property deletion feature. Please feel free to add examples of use cases.

**Comment by iltzortz** [comment]
Correct, i managed to do what i needed to do for the correction of the class identifier. Thank you for the hint. 

I still think though that a **Deletion feature** is necessary in more places - not only in Class/Properties definitions in order to encourage test/review workflows before you charactersize an ongoing namespace version as published.
e.g. i cannot delete a label that i created for testing, nor can i delete the entire project which i admit i created mainly for test purposes.

![image](https://github.com/geovistory/.github/assets/8971147/a2127684-3515-4bc4-a4ec-b8ee35359858)

Also, one might consider relevant the issue regarding hierarchical relationship deletion that i have described in detail here https://github.com/geovistory/ontome/issues/125
